drivers/sht1x/Kconfig

```
@@ -5,32 +5,53 @@
 # directory for more details.
 #
 
-if TEST_KCONFIG
+menuconfig MODULE_SHT1X
+    bool
+    prompt "SHT10/SHT11/SHT15 Humidity and Temperature Sensor"
```

Instead of the `select` we can do here:
```suggestion
    prompt "SHT10/SHT11/SHT15 Humidity and Temperature Sensor" if !(HAVE_SHT1X && MODULE_SAUL_DEFAULT)
    default y if (HAVE_SHT1X && MODULE_SAUL_DEFAULT)
```

---

done